packages/react-dom/src/__tests__/ReactRenderDocument-test.js

```diff
@@ -203,9 +203,9 @@ describe('rendering React components at document', () => {
             'with ReactDOM.hydrate() if you want React to attach to the server HTML.',
           {withoutStack: true},
         );
-      }).toWarnDev('Warning: Text content did not match.', {
-        withoutStack: true,
-      });
+      }).toWarnDev(
+        'Warning: Text content did not match. Server: "Goodbye world" Client: "Hello world"',
```

Comment:
Here both texts in the warning are added to the test. They are printed as quoted strings, the result of `JSON.stringify`.